pkg/server/jsonrpc.go

```diff
@@ -23,18 +23,24 @@ type JSONRPC interface {
 	communicate(buf []byte) (io.Reader, error)
 }
 
+// TODO: rename to remove unix since now this is generic (last commit)
 type unixSocketJSONRPC struct {
 	transport string
 	socket    *string
 	id        uint64
 }
 
 // NewUnixSocketJSONRPC creates a new instance of JSONRPC which is capable to
-// interact with unix domain socket
+// interact with either unix domain socket, e.g.: /var/tmp/spdk.sock
+// or with tcp connection ip and port tuple, e.g.: 10.1.1.2:1234
 func NewUnixSocketJSONRPC(socketPath string) JSONRPC {
```

Comment:
having 'socket' in the name is now obsolete as well, right? maybe sth like NewSpdk[Com|DataExchange|...]ChannelJSONRPC

Reply:
yes, exactly. just waiting for this to be merged first before change the name, since it will require PRs in other bridges to use new name.